src/messaging/ExchangeMgr.h

```diff
@@ -177,6 +178,8 @@ class DLL_EXPORT ExchangeManager : public SecureSessionMgrDelegate
 
     ReliableMessageMgr * GetReliableMessageMgr() { return &mReliableMessageMgr; };
 
+    MessageCounterSyncMgr * GetSecureChannelMgr() { return &mSecureChannelMgr; };
```

Comment:
Could we also rename the function and the associated member variables?

Reply:
Acked